MultiServer.py

```diff
@@ -30,13 +30,8 @@
     OperationalError = ConnectionError
 
 import NetUtils
-from worlds.AutoWorld import AutoWorldRegister
-
-proxy_worlds = {name: world(None, 0) for name, world in AutoWorldRegister.world_types.items()}
-from worlds import network_data_package, lookup_any_item_id_to_name, lookup_any_location_id_to_name
 import Utils
-from Utils import get_item_name_from_id, get_location_name_from_id, \
-    version_tuple, restricted_loads, Version
+from Utils import version_tuple, restricted_loads, Version
```

Comment:
`get_item_name_from_id` and `get_location_name_from_id` should be dead code now.
Maybe remove them? Those were the only place in Utils that imported worlds, so getting rid of those would reduce chance of accidental world import.

Reply:
They're still used in Tracker.py I believe, but as that's the only place now could probably be moved there.